.projenrc.ts

```diff
@@ -95,11 +95,10 @@ project.testTask.prependSpawn(installHelm);
 
 project.compileTask.spawn(schemas);
 
-// so that it works on windows as well
-// default projen uses $(npm pack) which fails
-project.packageTask.reset();
-project.packageTask.exec('mkdir -p dist/js');
-project.packageTask.exec('npm pack --pack-destination dist/js');
+// Allow skipping tests in build based on an env variable
+// This is only used by the package integrity check running outside the repo
+// While this check needs to replicate the release, it does not need to run tests
+project.testTask.addCondition("node -e \"if (process.env.SKIP_TESTS==='1') process.exit(1)\"");
```

Comment:
Wait shouldn't exit with zero?

Reply:
No that's right. It's just how projen conditions work:
- `Status==0` => success => run command
- `Status!=0` => failure => skip command
<img width="630" alt="image" src="https://github.com/user-attachments/assets/81ecb982-9989-4b33-9a5d-7a302ad51f28" />